modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteCacheObjectPutSelfTest.java

```diff
@@ -38,9 +38,9 @@
     @Override protected IgniteConfiguration getConfiguration(String gridName) throws Exception {
         IgniteConfiguration cfg = super.getConfiguration(gridName);
 
-        CacheConfiguration ccfg = new CacheConfiguration(DEFAULT_CACHE_NAME);
+        CacheConfiguration ccfg = new CacheConfiguration(CACHE_NAME);
 
-        ccfg.setName(CACHE_NAME);
+        ccfg.setOnheapCacheEnabled(true);
```

Comment:
Did you add this for some reason? Should we test heap instead of off-heap cases?